src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visitbalances/VisitBalanceResourceIntTest.kt

```diff
@@ -621,6 +650,17 @@ class VisitBalanceResourceIntTest : IntegrationTestBase() {
           .expectStatus().isOk
           .expectBody().isEmpty
       }
+
```

Comment:
This endpoint is used for reconciliation and already gracefully returns an empty body for the "dodgy offender"